Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-query-string-query.adoc

= Query String Query

A _query string_ can be used, to express a given query by means of a special syntax.

[source,json]
----
{ "query": "+nice +view" }
----

> A query string Query is demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of a query string Query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: updated
